locales/de/subtitles/vincent-pascucci/vincent-pascucci-02.yml

```diff
@@ -9,34 +9,32 @@ de:
           s: 1
           from: 00:00:01,22
           to: 00:00:11,17
-          text: During the elections of 1935, the reactionary government of Nanterre
+          text: Bei den Wahlen von 1935 wurde die reaktionäre Stadtverwaltung von Nanterre
         sequence-2:
           s: 2
           from: 00:00:11,17
           to: 00:00:22,19
-          text: had been replaced by a communist government.
+          text: durch eine kommunistische Regierung ersetzt.
         sequence-3:
           s: 3
           from: 00:00:22,19
           to: 00:00:32,07
-          text: During the electoral campaign, there were always meetings in the various
-            neighborhoods.
+          text: Während des Wahlkampfes gab es immer Versammlungen in den verschiedenen Vierteln.
         sequence-4:
           s: 4
           from: 00:00:32,07
           to: 00:00:41,19
-          text: These meetings were called “Le compte rendu des mandats”. My father
-            often took me along.
+          text: Diese Versammlungen hießen „Le compte rendu des mandats“. Mein Vater nahm mich oft mit.
         sequence-5:
           s: 5
           from: 00:00:41,19
           to: 00:00:48,17
-          text: I participated at meetings for the legislative elections
+          text: Ich nahm an Versammlungen zur Parlamentswahl statt,
```

Comment:
Kann man wirklich an Versammlungen **statt** nehmen? I still think, that `Ich nahm an Versammlungen zur Parlamentswahl teil` would be better.